data/json/items/armor/ballistic_armor.json

```diff
@@ -27,7 +27,7 @@
       }
     ],
     "price": "1600 USD",
-    "price_postapoc": "10 USD",
+    "price_postapoc": "50 USD",
```

Comment:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/POSTAPOC_PRICE_GUIDE.md 

You should read the apoc price guide, these price changes seem outrageously high.

Maybe there can be a markup for factions selling them, or a markup for militant factions buying them but what are the artisans gonna do with 14 ballistic vests.

Reply:
Prices adjusted